jena-patch/src/main/java/eu/neverblink/jelly/convert/jena/patch/JellyPatchOperations.java

```diff
@@ -0,0 +1,68 @@
+package eu.neverblink.jelly.convert.jena.patch;
+
+import com.google.protobuf.ExperimentalApi;
+import eu.neverblink.jelly.core.patch.PatchHandler;
+import eu.neverblink.jelly.core.proto.v1.PatchStatementType;
+import org.apache.jena.graph.Node;
+import org.apache.jena.rdfpatch.RDFChanges;
+
+/**
+ * Jelly-based operations on RDFChanges streams and RDFPatch objects from Jena.
+ */
+@ExperimentalApi
+public class JellyPatchOperations {
```

Comment:
Please keep it as `JellyPatchOps`. I'm actually reusing a convention from Jena in naming it like this.